pkg/virt-api/rest/subresource.go

```diff
@@ -269,6 +269,46 @@ func getRunningJson(vm *v1.VirtualMachine, running bool) string {
 	}
 }
 
+func (app *SubresourceAPIApp) patchVmStart(vm *v1.VirtualMachine, namespace string, startPaused bool) error {
+	patchType := types.MergePatchType
+
+	if startPaused {
+		patchString := "{\"spec\": {\"template\": {\"spec\": {\"startStrategy\":  \"Paused\"}}}}"
+		log.Log.Object(vm).V(4).Infof("Patching VM: %s", patchString)
+		_, err := app.virtCli.VirtualMachine(namespace).Patch(vm.GetName(), patchType, []byte(patchString))
```

Comment:
we can't perform two PATCH operations because we can't handle errors if one fails.

For example. lets say this Patch succeeds, but the next Patch fails... now we're left in a weird state where half the data for the start request is persisted permanently, and the other half never took hold.